{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:debian:python-imaging:-:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:python:pillow:2.3.0:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:python:pillow:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "2.3.1",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:python:pillow:2.5.0:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:python:pillow:2.5.1:*:*:*:*:*:*:*",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:python:pillow:2.5.2:*:*:*:*:*:*:*",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      },
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:o:opensuse:opensuse:13.2:*:*:*:*:*:*:*",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2014-3589"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "product": {
              "product_data": [
                {
                  "product_name": "python-imaging",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "=",
                        "version_value": "-"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "debian"
          },
          {
            "product": {
              "product_data": [
                {
                  "product_name": "pillow",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "=",
                        "version_value": "2.3.0"
                      },
                      {
                        "version_affected": "\u003c=",
                        "version_value": "2.3.1"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "2.5.0"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "2.5.1"
                      },
                      {
                        "version_affected": "=",
                        "version_value": "2.5.2"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "python"
          },
          {
            "product": {
              "product_data": [
                {
                  "product_name": "opensuse",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "=",
                        "version_value": "13.2"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "opensuse"
          }
        ]
      }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "PIL/IcnsImagePlugin.py in Python Imaging Library (PIL) and Pillow before 2.3.2 and 2.5.x before 2.5.2 allows remote attackers to cause a denial of service via a crafted block size."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-20"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "openSUSE-SU-2015:0798",
          "refsource": "SUSE",
          "tags": [],
          "url": "http://lists.opensuse.org/opensuse-updates/2015-04/msg00056.html"
        },
        {
          "name": "59825",
          "refsource": "SECUNIA",
          "tags": [],
          "url": "http://secunia.com/advisories/59825"
        },
        {
          "name": "DSA-3009",
          "refsource": "DEBIAN",
          "tags": [],
          "url": "http://www.debian.org/security/2014/dsa-3009"
        },
        {
          "name": "https://github.com/python-pillow/Pillow/commit/205e056f8f9b06ed7b925cf8aa0874bc4aaf8a7d",
          "refsource": "CONFIRM",
          "tags": [
            "Patch"
          ],
          "url": "https://github.com/python-pillow/Pillow/commit/205e056f8f9b06ed7b925cf8aa0874bc4aaf8a7d"
        },
        {
          "name": "https://pypi.python.org/pypi/Pillow/2.3.2",
          "refsource": "CONFIRM",
          "tags": [
            "Patch"
          ],
          "url": "https://pypi.python.org/pypi/Pillow/2.3.2"
        },
        {
          "name": "https://pypi.python.org/pypi/Pillow/2.5.2",
          "refsource": "CONFIRM",
          "tags": [
            "Patch"
          ],
          "url": "https://pypi.python.org/pypi/Pillow/2.5.2"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "accessComplexity": "LOW",
        "accessVector": "NETWORK",
        "authentication": "NONE",
        "availabilityImpact": "PARTIAL",
        "baseScore": 5,
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "vectorString": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
        "version": "2.0"
      },
      "exploitabilityScore": 10,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainOtherPrivilege": false,
      "obtainUserPrivilege": false,
      "severity": "MEDIUM",
      "userInteractionRequired": false
    }
  },
  "lastModifiedDate": "2018-10-30T16:27Z",
  "publishedDate": "2014-08-25T14:55Z"
}